packages/components/src/feature_alert/index.module.scss

```diff
@@ -0,0 +1,92 @@
+@import 'nebenan-ui-kit';
+
+$feature-alert-arrow-size: 16px;
+
+@keyframes feature-alert-appear {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+.tooltip {
+  width: 300px;
+  position: absolute;
+  cursor: pointer;
+  z-index: 1;
+
+  animation: 300ms ease-in feature-alert-appear;
+  visibility: hidden;
+
+  &.isActive {
+    visibility: visible;
```

Comment:
Please check if using visibility could lead to other implications on the page. I think it should be fine since the tooltip is absolute positioned but better to check

Reply:
It's actually has problems if using `display` prop - it breaks the tooltip positioning

Reply:
Yes but visibility has other implications - it's more like 'make this element invisible' - don't know how z-index works in that case